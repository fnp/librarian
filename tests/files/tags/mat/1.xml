<akap>
  Oznaczmy odległość tych miejsc przez δ,
  dalej odległość kliszy od kondensatora przez <wyroznienie>h</wyroznienie>,
  długość płyt metalowych kondensatora przez <wyroznienie>l</wyroznienie>,
  natężenie pola elektryczności przez <wyroznienie>F</wyroznienie>,
  masę elektronu przez <wyroznienie>m</wyroznienie>,
  jego ładunek przez <wyroznienie>e</wyroznienie>,
  a prędkość przez <wyroznienie>v</wyroznienie>.
  Możemy wyprowadzić wzór następujący
  <mat><mrow>
    <mi>δ</mi>
    <mo>=</mo>
    <mfrac>
      <mrow>
        <mi>e</mi>
        <mo>⁢</mo>
        <mi>F</mi>
        <mo>⁢</mo>
        <mi>l</mi>
        <mo>×</mo>
        <mfenced><mrow>
          <mfrac>
            <mi>l</mi>
            <mn>2</mn>
          </mfrac>
          <mo>+</mo>
          <mi>h</mi>
        </mrow>
        </mfenced>
      </mrow>
      <mrow>
        <mi>m</mi>
        <mo>⁢</mo>
        <msup>
          <mi>v</mi>
          <mn>2</mn>
        </msup>
      </mrow>
    </mfrac>
  </mrow></mat>
</akap>
