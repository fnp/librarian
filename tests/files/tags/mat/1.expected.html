<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Oznaczmy odległość tych miejsc przez δ,
  dalej odległość kliszy od kondensatora przez <em class="author-emphasis">h</em>,
  długość płyt metalowych kondensatora przez <em class="author-emphasis">l</em>,
  natężenie pola elektryczności przez <em class="author-emphasis">F</em>,
  masę elektronu przez <em class="author-emphasis">m</em>,
  jego ładunek przez <em class="author-emphasis">e</em>,
  a prędkość przez <em class="author-emphasis">v</em>.
  Możemy wyprowadzić wzór następujący
  <math xmlns="http://www.w3.org/1998/Math/MathML"><mrow>
    <mi>δ</mi>
    <mo>=</mo>
    <mfrac>
      <mrow>
        <mi>e</mi>
        <mo>⁢</mo>
        <mi>F</mi>
        <mo>⁢</mo>
        <mi>l</mi>
        <mo>×</mo>
        <mfenced><mrow>
          <mfrac>
            <mi>l</mi>
            <mn>2</mn>
          </mfrac>
          <mo>+</mo>
          <mi>h</mi>
        </mrow>
        </mfenced>
      </mrow>
      <mrow>
        <mi>m</mi>
        <mo>⁢</mo>
        <msup>
          <mi>v</mi>
          <mn>2</mn>
        </msup>
      </mrow>
    </mfrac>
  </mrow></math>
</p>
</div>
